Polygon_mesh_processing/doc/Polygon_mesh_processing/dependencies

```diff
@@ -12,3 +12,5 @@ Surface_mesh_deformation
 AABB_tree
 Triangulation_2
 Spatial_sorting
+Generator
```

Comment:
@afabri can you comment on why this is needed?

Reply:
In order to get a link to ` Random`   for the named parameter [` random_seed`](https://doc.cgal.org/latest/Polygon_mesh_processing/group__pmp__namedparameters.html#PMP_random_seed)